src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForSimulation.cs

```diff
@@ -189,6 +189,8 @@ public IMoBiSimulation CloneSimulation(IMoBiSimulation simulationToClone)
             return null;
 
          var newSimulation = _cloneManager.CloneSimulation(simulationToClone).WithName(newName);
+         newSimulation.Model.WithName(newName);
+         newSimulation.Model.Root.WithName(newName);
```

Comment:
2 questions:
why not just `XY.Name = newName` ?
when a simulation is _renamed_ - this was already implemented?

Reply:
Ok, using the command, but not adding to the history now.